webroot/results/admin/export_public.php

```diff
@@ -157,6 +154,10 @@ function exportPublic ( $sources ) {
         echo '.';  # shows both Apache and the user that the script is doing stuff and not hanging
       }
     }
+    //Check if any errors while fetching data
+    if (($error = mysql_error()) !== '') {
```

Comment:
Fair enough, I can get over that. I cannot, however, get over the idea that you're more concerned with getting email working. That's depressing :(